etc/config_base.sh

```diff
@@ -1,5 +1,7 @@
 # Script file included by all setup scripts to load local config.
 
+source etc/FILES_MAPPING
```

Comment:
Putting this here won’t work with the Debian package files locations since this script is also run and the env vars will not be the right ones.